src/core/Global.ttslua

```diff
@@ -2949,7 +2953,6 @@ end
 function loadOptionPanelState(newOptions)
   for id, state in pairs(newOptions) do
     if optionPanel[id] ~= state then
-      optionPanel[id] = state
```

Comment:
This was removed because it is already part of the `applyOptionPanelChange()` function